test/jdk/tools/jpackage/share/AddLauncherTest.java

```diff
@@ -89,14 +90,17 @@ public void test() {
 
         new AdditionalLauncher("Baz2")
                 .setDefaultArguments()
+                .addRawProperties(Map.entry("description", "Baz2 Description"))
```

Comment:
How expected description is verified in the test?

Reply:
I do not think that we have ability to check description on executable files. I added it for manual verification.

Reply:
Powershell can be used to extract the description from an executable. E.g.:
```
$ powershell -NoLogo -NoProfile -Command "(Get-Item build\\windows-x64\\images\\jdk\\bin\\java.exe).VersionInfo | select FileDescription"

FileDescription
---------------
Java(TM) Platform SE binary
```

You can create jdk.jpackage.test.AdditionalLauncher.verifyDescription() function that will call this script and call this function from jdk.jpackage.test.AdditionalLauncher.verify().